client/state/data-layer/wpcom/read/recommendations/sites/test/index.ts

```diff
@@ -3,35 +3,74 @@ import {
 	requestRecommendedSites as requestRecommendedSitesAction,
 	receiveRecommendedSites,
 } from 'calypso/state/reader/recommended-sites/actions';
-import { requestRecommendedSites, addRecommendedSites, fromApi } from '../';
+import {
+	requestRecommendedSites,
+	addRecommendedSites,
+	mapResponseToRecommendedSites,
+	RecommendedSitesBody,
+	RecommendedSite,
+} from '../index';
 
 const algorithm = 'chicken-recs/es1';
 const seed = 42;
-const response = {
+const response: RecommendedSitesBody = {
 	algorithm,
 	sites: [
 		{
 			blog_id: 19096129,
-			feed_id: 185124,
 			blog_title: 'Bente Haarstad Photography&amp;',
 			blog_url: 'http://bentehaarstad.wordpress.com',
+			description: 'Description 1',
+			feed_id: 185124,
+			feed_url: 'http://bentehaarstad.wordpress.com/feed/',
+			icon: {
+				ico: 'http://bentehaarstad.wordpress.com/favicon.ico',
+				img: 'http://bentehaarstad.wordpress.com/favicon.ico',
+				media_id: '12345',
+			},
+			ID: 12345,
+			name: 'Bente Haarstad Photography&amp;',
```

Comment:
Adding missing response fields to ensure that our implementation matches with the actual API.